plugins/speculation-rules/tests/test-speculation-rules-settings.php

```diff
@@ -122,4 +122,152 @@ public function test_plsr_add_settings_action_link(): void {
 			plsr_add_settings_action_link( $default_action_links )
 		);
 	}
+
+	/**
+	 * @covers ::plsr_get_mode_labels
+	 * @covers ::plsr_get_eagerness_labels
+	 * @covers ::plsr_get_setting_default
+	 * @covers ::plsr_register_setting
+	 */
+	public function test_register_settings(): void {
```

Comment:
I think better name this `test_plsr_register_setting` in accordance with the primary method it tests.
```suggestion
	public function test_plsr_register_setting(): void {
```